x-pack/test/functional/services/uptime.ts

```diff
@@ -8,12 +8,39 @@ import { KibanaFunctionalTestDefaultProviders } from '../../types/providers';
 
 export const UptimeProvider = ({ getService }: KibanaFunctionalTestDefaultProviders) => {
   const testSubjects = getService('testSubjects');
-
+  const wait = (seconds: number) => new Promise(r => setTimeout(r, seconds * 1000));
   return {
     async assertExists(key: string) {
       if (!(await testSubjects.exists(key))) {
         throw new Error(`Couldn't find expected element with key "${key}".`);
       }
     },
+    async navigateToPlugin() {
+      await testSubjects.click('homeSynopsisLinkuptime');
+    },
+    async setStartOfDatePickerRange(dateRangeStart: string) {
```

Comment:
The last Jenkins failure in these tests showed the timepicker still open with the cursor in the from time field.  If this is a new eui component we need to figure out if there's some loading indicator to wait for.

I don't think there should be datapicker methods in an uptime service file.  These should be common methods for the timepicker available to all plugins.   Maybe these methods are appropriate?
https://github.com/elastic/kibana/blob/master/test/functional/page_objects/header_page.js#L119

Reply:
Sorry, now I realize this is the new timepicker that isn't in use everywhere yet.  We still need to take a look to see if there's any loading indicators in the timepicker we need to check for.  And it should probably be a global service since it's going to be replacing the old timepicker everywhere in Kibana.

Reply:
@LeeDr want me to create an issue to that effect?

Reply:
@justinkambic What I think I'd like here is if you could move the timepicker functions to test/functional/services/timepicker.ts
And if you could change the name of `setStartOfDatePickerRange` to `setFromTime` to match the existing function here; https://github.com/elastic/kibana/blob/master/test/functional/page_objects/header_page.js#L119

But I'd like @dmlemeshko @silne30 and/or @spalger to weigh in on it too.  I think we'll need to implement more methods in that timepicker service very very soon.